edx_rest_framework_extensions/auth/jwt/authentication.py

```diff
@@ -17,6 +18,11 @@
 
 
 class CSRFCheck(CsrfViewMiddleware):
+
+    def __init__(self):
```

Comment:
Please check this.

Reply:
What should I check specifically? I think there is not Any dummy response issue.